src/java.desktop/share/classes/javax/swing/filechooser/FileSystemView.java

```diff
@@ -255,6 +256,54 @@ public Icon getSystemIcon(File f) {
         }
     }
 
+   /**
+    * Returns an icon for a file, directory, or folder as it would be displayed
+    * in a system file browser for the requested size.
+    * <p>
+    * The default implementation gets information from the
+    * {@code ShellFolder} class. Whenever possible, the icon
+    * returned will be a multi-resolution icon image,
+    * which will allow better scaling with different
+    * magnification factors.
```

Comment:
I'm not sure what are the standard terms to refer to High DPI environments and the scale factors associated with High DPI.

Reply:
In Windows environment it is scaling size or scaling factor. On Mac it is called scaled resolution. So i would say scaling factor is an appropriate term.